<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: rgb(3, 4, 19);
border: 2px solid rgb(121, 121, 121);
}</string>
  </property>
  <widget class="QPushButton" name="NextBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color:  #252984;
color: white;
font: bold 25px;</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color:  #252984;
color: white;
font: bold 25px;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel 
{
	color: white;
	font: bold 25px;
}</string>
   </property>
   <property name="text">
    <string>Set width and height for each rectangle</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RectLine">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background: white;
	font: bold 25px;
    border: 3px solid #797979;
	color: black;
}

QLineEdit[text=&quot;&quot;]{
	background: white;
	font: bold 25px;
    border: 3px solid #797979;
	color: black;
}
</string>
   </property>
   <property name="placeholderText">
    <string>W1 H1 W2 H2 W3 H3...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
